README.md

```diff
@@ -1,15 +1,17 @@
-# Project 3: Spots
+# Project 9: Spots
 
```

Comment:
* Please, delete the dupilcated files from the project https://snipboard.io/v6I4YO.jpg